crates/ruff_linter/resources/test/fixtures/pycodestyle/E30.py

```diff
@@ -411,17 +411,9 @@ async def function1():
 
 # no error
 async def function1():
-	await function2()
+    await function2()
     async with function3():
-    	pass
-# end
-
-
-# no error
-class Test:
-    async
```

Comment:
Do you think it would make sense to replace this test with another soft keyword like `type` or `match`?